backend/pkg/logger/errors.go

```diff
@@ -0,0 +1,32 @@
+package logger
+
+import (
+	"time"
+
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
+)
+
+type ErrLoggerNotFound struct {
+	name abstraction.LoggerName
+}
+
+func (e *ErrLoggerNotFound) Error() string {
+	return "Logger not found: " + string(e.name)
+}
+
+type ErrCreatingFile struct {
+	name      abstraction.LoggerName
+	timestamp time.Time
+}
+
+func (e *ErrCreatingFile) Error() string {
+	return "Error creating file for logger " + string(e.name) + " at " + e.timestamp.Format(time.RFC3339)
+}
+
+func NewLoggerNotFoundError(name abstraction.LoggerName) error {
+	return &ErrLoggerNotFound{name: name}
+}
+
+func NewErrCreatingFile(name abstraction.LoggerName, timestamp time.Time) error {
+	return &ErrCreatingFile{name: name, timestamp: timestamp}
+}
```

Comment:
Instead make the fields on the erros public so everyone can access. This way there is no need to have a constructor for the values as you can directly construct the error where you need it